src/ast/ExecutionOrder.h

```diff
@@ -10,15 +10,14 @@
  *
  * @file ExecutionOrder.h
  *
- * Defines AST Clauses
+ * Defines the execution order class
  *
  ***********************************************************************/
 
 #pragma once
 
+#include "Node.h"
 #include "SrcLocation.h"
-#include "ast/Node.h"
```

Comment:
This will still compile, but it is more consistent to refer to classes using their path within the src directory. Apart from reducing the introduction of errors when files are moved or includes changed, the path is now part of the description of the file to be included and this change reduces that information content.

If you really want to change the way we include headers you should also change all the other souffle files in subdiectories to do so as well, for consistency. Exceptions may need to be made for all files with the same name, such as `Relation.h` and `Relation.h` - that can cause issues in some cases.

Reply:
I was not sure about this because I saw inconsistencies. I am happy to remove all ast/ prefices in the includes.

Reply:
and make the includes directory local.